src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+    rel="preconnect" 
+    href="https://fonts.gstatic.com" 
+    crossorigin="fonts"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel+Decorative:wght@400;700&family=Cinzel:wght@600;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyoBox">
+      <nav class="box text lastBox active">
+        Video
+      </nav>
+      <nav class="box text active">
+        Photo
+      </nav>
+      <nav class="box text active">
+        Tablets
+      </nav>
+      <nav class="box text active">
+        Gadgets
+      </nav>
+      <nav 
+      class="box text active"
+      data-qa="hover"
+      >
```

Comment:
```suggestion
      <nav 
        class="box text active"
        data-qa="hover"
      >
```
fix code style everywhere